Code/Editor/Plugins/ComponentEntityEditorPlugin/SandboxIntegration.cpp

```diff
@@ -691,11 +697,27 @@ void SandboxIntegrationManager::PopulateEditorGlobalContextMenu(QMenu* menu, con
         SetupSliceContextMenu(menu);
     }
 
-    action = menu->addAction(QObject::tr("Duplicate"));
-    QObject::connect(action, &QAction::triggered, action, [this] { ContextMenu_Duplicate(); });
-    if (selected.size() == 0)
+    if (!selected.empty())
     {
-        action->setDisabled(true);
```

Comment:
This change moves it from disabling the option to not having it in the list at all.  Was this change passed through UX?

Reply:
I discussed this topic with @rainbj some time ago and the determination was that actions that should not be possible should be removed instead of disabled in the context menu. The conversation wasn't specifically related to read-only entities though, so we may re-discuss this in this context if we believe disabling options makes more sense.